src/app/core/common-components/entity-subrecord/list-paginator/list-paginator.component.ts

```diff
@@ -53,33 +43,24 @@ export class ListPaginatorComponent<E> implements OnChanges, OnInit {
 
   onPaginateChange(event: PageEvent) {
     this.pageSize = event.pageSize;
-    this.updateUserPaginationSettings();
+    this.savePageSize(this.pageSize);
   }
 
-  private async applyUserPaginationSettings() {
-    if (!this.user) {
-      return;
-    }
-
-    const savedSize =
-      this.user?.paginatorSettingsPageSize[this.idForSavingPagination];
+  private applyUserPaginationSettings() {
+    const savedSize = this.getSavedPageSize();
     this.pageSize = savedSize && savedSize !== -1 ? savedSize : this.pageSize;
   }
 
-  private async updateUserPaginationSettings() {
-    if (!this.user) {
-      return;
-    }
-    // The page size is stored in the database, the page index is only in memory
```

Comment:
This behaviour has already been removed a while ago. Should we add this, maybe stored in [session storage](https://stackoverflow.com/a/19869560/10713841)?

Reply:
in my opinion, page index doesn't have to be persisted. Although maybe for navigating back after looking at a details view, it may be nice to keep that somehow. If it's not there now, I think I wouldn't invest time here to implement it.